### PR TITLE
blui 3927 update verbiage translations

### PR DESCRIPTION
**Pull request by JeffGreiner-eaton**
Fixes BLUI-3927.

#### Changes proposed in this Pull Request:

- Update VERIFICATION_CODE_PROMPT & ACTIONS.RESEND verbiage
-
-

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

- run workflow branch https://github.com/etn-ccis/blui-react-workflows/pull/241 or dev if merged
- cd shared-auth and checkout feature/blui-3927-update-verbiage-prompts
- yarn start:example

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Review by huayunh — APPROVED** [review]

**Comment by huayunh** [comment]
Need to update changelog and package version

File: src/data/translations/english.ts [hunk 1]
```diff
@@ -80,7 +80,7 @@ const resources: LanguageFile = {
                 MESSAGE:
                     'A verification code has been sent to the email address you provided. Click the link or enter the code below to continue. This code is valid for 30 minutes.',
                 RESEND: 'Resend Verification Email',
-                VERIFICATION_CODE_PROMPT: "Didn't receive a code?",
+                VERIFICATION_CODE_PROMPT: "Didn't receive email?",
```
Comment (daileytj) [inline src/data/translations/english.ts:83]:
```suggestion
                VERIFICATION_CODE_PROMPT: "Didn't receive an email?",
```

Comment (daileytj) [reply src/data/translations/english.ts:83]: Not sure how other languages handle the indefinite article, but may need to update the translations as well.

Comment (JeffGreiner-eaton) [reply src/data/translations/english.ts:83]: I matched what was approved for NG workflow translations, but yes it should be Didn't receive an email? 
either way is fine for me.

Comment (JeffGreiner-eaton) [reply src/data/translations/english.ts]: @daileytj  & @huayunh updated to include 'an' in english and other transalations

**Review by daileytj — APPROVED** [review]

**Review by huayunh — CHANGES_REQUESTED** [review]
Changelog?

**Comment by daileytj** [comment]
> Changelog?

May want to add a beta package version to the package.json too so you can publish and use this.

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "@brightlayer-ui/react-auth-shared",
     "description": "Re-usable react logic for Authentication and Registration within Eaton applications. For use with @brightlayer-ui/react-native-auth-workflow and @brightlayer-ui/react-auth-workflow.",
-    "version": "3.7.4",
+    "version": "3.7.5",
```
Comment (daileytj) [inline package.json:4]:
```suggestion
    "version": "3.7.5-beta.0",
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v3.7.5 (Unreleased)
+
+### Changed
+
+-   Changed verbiage and translations on verify email registration resend code.
```
Comment (huayunh) [inline CHANGELOG.md:12]: May want to ensure that the changelog reads like a sentence

```suggestion
-   Changed resend email button's text label on the email verification screen.
```

**Review by huayunh — APPROVED** [review]
Let's go!

**Pull request merged**